### PR TITLE
Skylake Graphics spoofing support for macOS 13+

### PR DESCRIPTION
**Pull request by PMheart**
CC and thanks to @dhinakg

### CODE REVIEW

File: Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+	<key>CFBundleVersion</key>
+	<string>1.0.0</string>
+	<key>IOKitPersonalities</key>
+	<dict>
+		<key>Gen7</key>
+		<dict>
+			<key>CFBundleIdentifier</key>
+			<string>com.apple.driver.AppleIntelKBLGraphics</string>
+			<key>Development</key>
+			<dict>
```
Comment (vit9696) [inline Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist:24]: What values are different from KBL kext here?

Comment (PMheart) [reply Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist:24]: CC @dhinakg

Comment (dhinakg) [reply Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist:24]: Well, it is easier to mention the differences from the SKL kext: 
- `Development/DisablePageFaultHandling` and `Development/DisplaySurfaceInEDRAM` were added from the KBL kext (was unsure whether they mattered, so just left them in)
- all the bundle names are changed to the KBL ones
- `IOVARendererID` is now the value from the KBL kext. 
I figured copying the values from the SKL kext would provide the most compatibility; although it's possible they dropped the old stuff, I haven't noticed any breakage yet.

The differences from the KBL kext:
- `Gen95` references changed to `Gen9`
- **`IOGVAXDecode` removed** (this was the cause of the YouTube issues - Skylake does not have VP9 hw decode)
- `IOGVAHEVCDecode` changed to `1` (KBL was `2`)
- Profile 2 removed from `IOGVAHEVCDecodeCapabilities` and `IOGVAHEVCEncodeCapabilities`
- `VTRating` in `IOGVAHEVCEncodeCapabilities` changed to `90` (KBL was `100`)

I must admit I don't know the significance of most of these changes but given they are a straight rip from the SKL kext it *should* work

File: Manual/FAQ.IntelHD.en.md [hunk 2]
```diff
@@ -1044,8 +1044,12 @@ Mobile: 0, PipeCount: 2, PortCount: 2, FBMemoryCount: 2
   
 ## Intel HD Graphics 510-580 ([Skylake](https://en.wikipedia.org/wiki/Skylake_(microarchitecture)))  
 
-> Supported since OS X 10.11.4 to macOS 12.x. On newer operating systems these are not supported.  
-  
+> Officially supported since OS X 10.11.4 to macOS 12.x. On newer operating systems, spoofing as Kaby Lake is required.
+
+#### Spoof Skylake as Kaby Lake on macOS Ventura (13) and above
+
+First, make sure that WhateverGreen v1.6.0 and above is used. Then, use [SKLAsKBLGraphicsInfo.kext](https://github.com/acidanthera/WhateverGreen/tree/master/Manual/Kexts/SKLAsKBLGraphicsInfo.kext) by loading it with the bootloader. Eventually, it is necessary to fake `device-id` and choose an `ig-platform-id` from Kaby Lake that is closest to the Skylake model (e.g. HD 530 to HD 630).
```
Comment (PMheart) [inline Manual/FAQ.IntelHD.en.md:1051]: TODO: Drop `SKLAsKBLGraphicsInfo.kext `

File: Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist [hunk 3]
```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (PMheart) [inline Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist:1]: TODO: Drop this injector

File: WhateverGreen/kern_igfx.cpp [hunk 4]
```diff
@@ -61,12 +61,27 @@ void IGFX::init() {
...
 		case CPUInfo::CpuGeneration::Skylake:
-			supportsGuCFirmware = true;
-			currentGraphics = &kextIntelSKL;
-			currentFramebuffer = &kextIntelSKLFb;
-			modForceCompleteModeset.supported = modForceCompleteModeset.legacy = true; // not enabled, as on legacy operating systems it casues crashes.
-			modTypeCCheckDisabler.enabled = getKernelVersion() >= KernelVersion::BigSur;
-			modBlackScreenFix.available = true;
+			// Fake SKL as KBL on 13.0+ due to the removal of SKL kexts
+			// Or KBL kext can be used on SKL with older versions as well
+			forceSKLAsKBL = checkKernelArgument("-igfxsklaskbl") || getKernelVersion() >= KernelVersion::Ventura;
```
Comment (PMheart) [inline WhateverGreen/kern_igfx.cpp:66]: TODO: Docs, README, and Changelog

File: Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist (hunk 1 above)
Comment (PMheart) [reply Manual/Kexts/SKLAsKBLGraphicsInfo.kext/Contents/Info.plist:24]: UPDATE - Resolved/Spoiled due to the abolishing of this codeless kext.

**Pull request merged**